pkg/proxy/node_test.go

```diff
@@ -17,13 +17,21 @@ limitations under the License.
 package proxy
 
 import (
+	"k8s.io/klog/v2"
```

Comment:
This import should be align with other `k8s.io' package.

Reply:
done.